src/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/ReadResourcesValidation.feature

```diff
@@ -51,47 +51,47 @@ Feature: Resources "Read" Operation validations
                   """
 
         Scenario: 03 Verify response code 404 when ID does not exist
-             When a GET request is made to "ed-fi/absenceEventCategoryDescriptors/123123123123"
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/123123123123"
              Then it should respond with 404
 
         Scenario: 04 Verify array records content on GET All
-             When a GET request is made to "ed-fi/absenceEventCategoryDescriptors"
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
              Then it should respond with 200
               And total of records should be 1
 
         Scenario: 05 Verify response code 404 when trying to get a school with an ID that corresponds to Course
-            Given the system has these "Schools" 
-                  | schoolId | nameOfInstitution | educationOrganizationCategories                                                                                         | gradeLevels                                                                          |
-                  | 100      | School Test       | [{ "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#School"}]   | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Ninth grade"}]      |
-            And a POST request is made to "courses" with
-                   """
-                   {
-                        "courseCode": "ALG-2",
-                        "identificationCodes": [
-                          {
-                            "courseIdentificationSystemDescriptor": "uri://ed-fi.org/CourseIdentificationSystemDescriptor#LEA course code",
-                            "identificationCode": "ALG-2"
-                          }
-                        ],
-                        "educationOrganizationReference": {
-                          "educationOrganizationId": 100
-                        },
-                        "courseTitle": "Algebra 2",
-                        "numberOfParts": 2
-                    }
-                   """
-             When a GET request is made to "ed-fi/schools/{id}"
+            Given the system has these "Schools"
+                  | schoolId | nameOfInstitution | educationOrganizationCategories                                                                                       | gradeLevels                                                                     |
+                  | 100      | School Test       | [{ "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#School"}] | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Ninth grade"}] |
+              And a POST request is made to "courses" with
```

Comment:
Do we need the leading slash `/courses` ?

Reply:
Technically no, but it should be failing since it does not have `ed-fi/`. Unfortunately, the mistake in line 66 results in a 404 - which is exactly what this test is looking for! A false positive.

Reply:
Fixed